eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import pluginReactConfig from "eslint-plugin-react/configs/recommended.js";
+
+
+export default [
+  {files: ["**/*.{js,mjs,cjs,jsx}"]},
+  { languageOptions: { parserOptions: { ecmaFeatures: { jsx: true } } } },
+  {languageOptions: { globals: globals.browser }},
```

Comment:
some inconsistent whitespaces around `[]` and `{}` :wink:

Reply:
do we have linting for this foreman_fog_proxmox project?